extra/changelog-generator/changelog-generator

```diff
@@ -36,14 +36,24 @@ def aggregate_dependabot_changelogs(entry_list):
                 # then strip start and end whitespace, but keep the rest of the
                 # whitespace intact for indentation and line separation
                 # purposes.
-                aggregated_dependabot_changelogs += "\n* " + "\n    ".join(
-                    [
-                        s
-                        for n, s in enumerate(
-                            entry.split("\n", 1)[1].strip().split("\n")
-                        )
-                    ]
-                )
+                try:
+                    aggregated_dependabot_changelogs += "\n* " + "\n    ".join(
+                        [
+                            s
+                            for n, s in enumerate(
+                                entry.split("\n", 1)[1].strip().split("\n")
+                            )
+                        ]
+                    )
+                except IndexError:  # it may happen that only first line is available
+                    aggregated_dependabot_changelogs += "\n* " + "\n    ".join(
+                        [
+                            s
+                            for n, s in enumerate(
+                                entry.split("\n", 1)[0].strip().split("\n")
```

Comment:
Q:
If only one line is available, why do we have to split this?

Reply:
Done, thanks!